exampleAudioSink/src/main/java/com/twilio/exampleaudiosink/WavFileHelper.java

```diff
@@ -0,0 +1,247 @@
+package com.twilio.exampleaudiosink;
+
+import android.content.Context;
+import android.media.AudioFormat;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.RandomAccessFile;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+public class WavFileHelper {
+
+    private static final String fileName = "/audio_sink.wav";
+    private static boolean didWriteWavHeader;
```

Comment:
I think `didWriteWavHeader` and `didCompleteWavHeader` should not be marked as `static`